MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+
+You can display the complex type property values in combo box editor by using the DisplyMemberPath and SelectedValuePath properties of DataFormComboBoxItem. You need to use `GenerateDataFormItem `event to set `DisplayMemberPath` and `SelectedValuePath` property value of DataFormComboBoxItem for complex type property.
+
+You can display the complex type property values in autocomplete editor by using the DisplayMemberPath and SelectedValuePath properties of DataFormAutoCompleteItem. Use the `GenerateDataFormItem `event to set `DisplayMemberPath` and `SelectedValuePath` property values of AutoComplete for complex type property.
+
+N> Class cannot be directly set as data type for autocomplete editor in this complex type scenario.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.DataObject = new ContactModel();
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+    dataForm.RegisterEditor("EmployeeDetails", DataFormEditorType.ComboBox);
+ 
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "EmployeeDetails" && e.DataFormItem is DataFormAutoCompleteItem autoCompleteItem)
+        {   
+            autoCompleteItem.DisplayMemberPath = "ID";
+            autoCompleteItem.SelectedValuePath = "Name";
+        }
+    } 
+ 
+    public class DataFormItemsSourceProvider  : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            if (sourceName == "EmployeeDetails")
+            {
+                List<EmployeeInfo> details = new List<EmployeeInfo>();
+                details.Add(new EmployeeInfo() { ID = 1, Name =  "Eric"  });
+                details.Add(new EmployeeInfo() { ID = 2, Name = "James"  });
+                details.Add(new EmployeeInfo() { ID = 3, Name = "Jacob"  });
+                details.Add(new EmployeeInfo() { ID = 4, Name = "Lucas"  });
+                details.Add(new EmployeeInfo() { ID = 5, Name = "Mark"  });
+                details.Add(new EmployeeInfo() { ID = 6, Name = "Aldan"  });
+                details.Add(new EmployeeInfo() { ID = 7, Name = "Aldrin"  });
+                details.Add(new EmployeeInfo() { ID = 8, Name = "Alan"  });
+                details.Add(new EmployeeInfo() { ID = 9, Name = "Aaron"  });
+
+                return details;
+            }
+        return new List<string>();
+        }
+    }
+
+    public class ContactInfo
+    {
+        [Display(Name ="First Name")]
+        public String FirstName { get; set; } 
+        public string EmployeeDetails { get; set; }
+    }
+
+    public class EmployeeInfo
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+`Image`
+
+The `DataFormAutoCompleteEditor` provides display the filtered suggestions.
```

Comment:
Remove
The `DataFormAutoCompleteEditor` provides display the filtered suggestions.

The `AutoCompleteMode` property is used to choose the suggestion display mode in the `DataFormAutoCompleteItem` class.

Reply:
removed